test/blockquote_extra_quote_remover_test.rb

```diff
@@ -0,0 +1,94 @@
+# encoding: UTF-8
+
+require "test_helper"
+
+class BlockquoteExtraQuoteRemoverTest < Minitest::Test
+  def assert_remover_transforms(options)
+    from = options.keys.first
+    to = options.values.first
+    assert_equal to, Govspeak::BlockquoteExtraQuoteRemover.remove(from)
+  end
+
+  def assert_leaves_untouched(candidate)
+    assert_remover_transforms candidate => candidate
+  end
+
+  test "ignores nil" do
+    assert_leaves_untouched nil
+  end
+
+  test "ignores text without double quotes" do
+    assert_leaves_untouched %{no quotes\na few lines,\n\n\n\nbut no quotes\n\n}
+  end
+
+  test "ignores text without a quote symbol" do
+    assert_leaves_untouched %{quotes\n"but not with a > symbol"}
+  end
+
+  # TODO check this is actually behaviour we want
```

Comment:
I'm not sure really - I could probably phrase this note better to indicate that it's kept for consistency but this may not be desired behaviour into the future